lib/fluent/agent.rb

```diff
@@ -137,7 +131,12 @@ def add_match(type, pattern, conf)
       @outputs << output
       if output.respond_to?(:outputs) && (output.respond_to?(:multi_output?) && output.multi_output? || output.is_a?(Fluent::MultiOutput))
```

Comment:
Another topic.
I noticed adding `Fluent::MultiOutput`'s `outputs` seems wrong.
v0.12 `MultiOutput` calls plugin's `start`/`shutdown` directly, so
it causes double lifecycle sequence calls.